MetaMask/metamask-extension
# UX: Multichain: Send flow: Implement Textarea for Hex data

## **Description**

The amazing Design System team created a `<Textarea>` component which this PR employs.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Run `MULTICHAIN=1 yarn start`
2. Go to Advanced Settings -> Turn on Show Hex Data
3. Start the Send Flow
4. See the textarea

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="421" alt="SCR-20231214-noyv" src="https://github.com/MetaMask/metamask-extension/assets/46655/52a5c0d1-e78a-4c40-a57f-5b952594df5d">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
